### DISCUSSION TITLE
Conversa reaberta ou novo contato.

### DISCUSSION THREAD

**Discussion by MHC2023** [chatwoot/chatwoot#7444] [Feature Requests]
Toda vez que o mesmo cliente chama, o chat é reaberto com o mesmo número de protocolo e ainda com o nome do agente anterior. Assim causa um problema de controle de quantidade de chamados no chat, pois se o cliente chama varias vezes ele sempre tera o mesmo protocolo e também fica dificil para conseguir controlar quem é o responsavel pelo chamado pois sempre fica para o ultimo responsavel quando o cliente abre o chamado, dificultando verificar o historico de conversa. 
Poderia que cada vez que tivesse uma nova conversa, gera - se um novo protocolo e não ao mesmo prontocolo da conversa anterior. Assim cada conversa teria o seu protocolo facilitando de verificar o historico.

**Comment by OtavioACurioni** [comment]
Essa seria uma atualização muito relevante para o uso do chat.

**Comment by Brunotambarussi** [comment]
Essa funcionalidade seria essencial para que o chat melhorasse a experiência de todos os usuários.

**Comment by MarconiD7** [comment]
Atualmente está bem complicado controlar os chamados, SLA e etc. Essa funcionalidade seria MUITO importante.

**Comment by matiasduartee** [comment]
Realmente isso que @MHC2023 trouxe é muito chato.
